src/XMakeTasks/Microsoft.Common.CrossTargeting.targets

```diff
@@ -95,19 +95,16 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     </PropertyGroup>
   </Target>
 
-  <!--
-    This will import NuGet restore targets, which is a special case separate from the package -> project extension 
-    mechanism below. For obvious reasons,  we need restore to work before any package assets are available.
+  <Import Project="$(CustomAfterMicrosoftCommonCrossTargetingTargets)" Condition="'$(CustomAfterMicrosoftCommonCrossTargetingTargets)' != '' and Exists('$(CustomAfterMicrosoftCommonCrossTargetingTargets)')"/>
```

Comment:
Yes, this looks like it allow NuGet to get extra targets into the project to continue the walk.